<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="SOHANCHAVAN" failures="1" tests="2" name="Default test(failed)" time="1.128" errors="0" timestamp="2022-01-25T13:11:18 IST">
  <testcase classname="Test.LoginTest" name="test2" time="1.024">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method Test.LoginTest.test2() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method Test.LoginTest.test2() didn't finish within the time-out 1000
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="Test.LoginTest" name="god" time="0.0">
    <skipped/>
  </testcase> <!-- god -->
  <testcase classname="Test.LoginTest" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
